packages/core/test/unit/html/NodeProcessor.test.ts

```diff
@@ -122,6 +123,74 @@ test('processNode processes dropdown with header slot taking priority over heade
                            testData.PROCESS_DROPDOWN_HEADER_SLOT_TAKES_PRIORITY_EXPECTED);
 });
 
+test('processNode does not log warning when lazy pic has width or height',
+     () => {
+       const nodeProcessor = getNewDefaultNodeProcessor();
+
+       const testCode = '<pic scr="" alt="" width="300" lazy></pic>';
+       const testNode = parseHTML(testCode)[0] as MbNode;
+
+       const consoleSpy = jest.spyOn(logger, 'warn');
```

Comment:
I'm OK with this I think - though I remember some discussions about spy-on vs mock, where mock tends to be preferred over spying as errors can't be produced from winston. 

Here is some additional discussions on stack overflow for your consideration...
* [Using Jest to mock winston logger](https://stackoverflow.com/questions/57771616/how-to-use-jest-to-mock-winston-logger-instance-encapsulated-in-service-class)
* [Mocking vs spying](https://stackoverflow.com/questions/12827580/mocking-vs-spying-in-mocking-frameworks)

And a related issue in markbind #2099 - feel free to weigh in with your thoughts.

Thanks @luminousleek for sharing these links with me!

Reply:
I've got an example of how you might mock the logger in #2463

Reply:
Thank you for the suggestions @kaixin-hc @luminousleek ! Will follow the suggestions and update my code.